gooddata-java/src/test/java/com/gooddata/sdk/service/project/ProjectServiceIT.java

```diff
@@ -451,4 +454,23 @@ public void removeUserFromProjectNotFound() {
 
         gd.getProjectService().removeUserFromProject(enabled, account);
     }
+
+
+    @Test
+    public void shouldListProjectsForUser() {
+        onRequest()
+            .havingMethodEqualTo("GET")
+            .havingPathEqualTo(ProjectService.LIST_PROJECTS_TEMPLATE.expand("ID").toString())
+            .havingQueryStringEqualTo("offset=0&limit=100")
+            .respond()
+            .withStatus(200)
+            .withBody(readFromResource("/project/projects-for-user.json"));
+
+
+        final PageBrowser<Project> projects = gd.getProjectService().listProjects(
+            account,
+            new CustomPageRequest(0, 100));
+        assertThat(projects.getPageItems().isEmpty(), Is.is(false));
```

Comment:
Why do you import `org.hamcrest.core.Is`? There's already `org.hamcrest.Matchers.is` imported.

```suggestion
        assertThat(projects.getPageItems().isEmpty(), is(false));
```